src/selfplay/tournament.cc

```diff
@@ -37,6 +37,9 @@
 
 namespace lczero {
 namespace {
+const OptionId kAdjudicateId{"adjudicate", "Adjudicate",
```

Comment:
I would also prefer this to be `tb-adjudicate` to avoid confusion: we also have game length adjudication that is currently always on.